sdk/communication/Azure.Communication.MediaComposition.Tests/samples/Sample1_MediaCompositionClient.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Communication.MediaComposition.Models;
+
+namespace Azure.Communication.MediaComposition.Tests.samples
+{
+    /// <summary>
+    /// Basic Azure Communication.MediaComposition samples.
+    /// </summary>
+    public partial class Sample1_MediaCompositionClient
+    {
+        // TODO: Update the samples to tests when the E2E flow is working
+        private const string mediaCompositionId = "2x2Grid";
+        public async Task CreateMediaCompositionAsync()
+        {
+            var mediaCompositionClient = CreateMediaCompositionClient();
+
+            var layout = new MediaCompositionLayout()
+            {
+                Resolution = new LayoutResolution()
+                {
+                    Width = 1920,
+                    Height = 1080
+                },
+                Grid = new GridLayoutOptions(
+                    rows: 2,
+                    columns: 2,
+                    new string[2, 2] { { "jill", "jack" }, { "jane", "jerry" }
+                }),
+            };
+
+            var inputs = new Dictionary<string, MediaInput>()
+            {
+                {
+                    "jill",
+                    new MediaInput()
+                    { Participant = new ParticipantInput()
+                        { Id = new CommunicationIdentifierModel()
+                            { MicrosoftTeamsUser = new MicrosoftTeamsUserIdentifierModel("f3ba9014-6dca-4456-8ec0-fa03cfa2b7b7") },
+                            Call = "teamsMeeting",
+                            PlaceholderImageUri = "https://imageendpoint"
```

Comment:
I think placeholderImageUri should be option (if it already isn't)

Reply:
Yup, currently, it's an optional property and users can decide if they want to set it.